Wrappers/Python/cil/optimisation/functions/TotalVariation.py

```diff
@@ -54,6 +54,8 @@ class TotalVariation(Function):
       :type split: bool, default `False`           
       :param info: force a print to screen stating the stop
       :type info: bool, default `False`
+      :param strongly_convex_constant: Adds a strongly convex function to the TotalVariation functional.
```

Comment:
"a strongly convex function" is a bit vague - can you be more precise on what is added where.

Reply:
Is there a reference to include to describe the strategy of adding a strongly convex term?

Reply:
Could some suggestion be added on what size strongly convex parameter to choose? Around 0.1? Around 1, around 100,  1e10,    minus 1000?  

And briefly describe the effect and any tradeoffs in selecting it (eg a large value leads to faster convergence but a cruder approximation of the TV problem - or similar).

Reply:
- Fixed docs
- Reference added

How to choose the strong convexity parameter is an open question. Similar to how to choose the regularisation parameter. The only suggestion (added in docs) is to choose a relatively small parameter so it cannot act as an additional regulariser.